# Korean translation for update-manager
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the update-manager package.
# darehanl <darehanl@gmail.com>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2010-09-22 05:12+0000\n"
"PO-Revision-Date: 2010-09-24 08:54+0000\n"
"Last-Translator: Para <Unknown>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=1; plural=0;\n"
"X-Launchpad-Export-Date: 2010-09-27 08:04+0000\n"
"X-Generator: Launchpad (build Unknown)\n"

#. TRANSLATORS: download size is 0
#: ../DistUpgrade/utils.py:346 ../UpdateManager/Core/utils.py:346
msgid "0 KB"
msgstr "0 KB"

#. TRANSLATORS: download size of very small updates
#: ../DistUpgrade/utils.py:349 ../UpdateManager/Core/utils.py:349
msgid "1 KB"
msgstr "1 KB"

#. TRANSLATORS: download size of small updates, e.g. "250 KB"
#: ../DistUpgrade/utils.py:352 ../UpdateManager/Core/utils.py:352
#, python-format
msgid "%.0f KB"
msgstr "%.0f KB"

#. TRANSLATORS: download size of updates, e.g. "2.3 MB"
#: ../DistUpgrade/utils.py:355 ../UpdateManager/Core/utils.py:355
#, python-format
msgid "%.1f MB"
msgstr "%.1f MB"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:209 ../DistUpgrade/distro.py:424
#, python-format
msgid "Server for %s"
msgstr "%s 서버"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:227 ../DistUpgrade/distro.py:233
#: ../DistUpgrade/distro.py:249
msgid "Main server"
msgstr "주 서버"

#: ../DistUpgrade/distro.py:253
msgid "Custom servers"
msgstr "사용자 정의 서버"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:131
msgid "Could not calculate sources.list entry"
msgstr "sources.list 항목을 계산할 수 없습니다."

#: ../DistUpgrade/DistUpgradeAptCdrom.py:234
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr "패키지 파일을 찾을 수 없습니다. 우분투 디스크가 아니거나 잘못된 아키텍쳐(x86, x64)입니다."

#: ../DistUpgrade/DistUpgradeAptCdrom.py:277
msgid "Failed to add the CD"
msgstr "CD 추가하기 실패"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:278
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"CD를 더할 때 오류가 발생하였기 때문에 업그레이드는 중단될 것입니다. 올바른 우분투 CD를 사용하고 있었다면 이 버그를 보고해 "
"주십시오.\n"
"\n"
"오류 메시지:\n"
"'%s'"

#: ../DistUpgrade/DistUpgradeCache.py:130
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "오류 상태인 패키지 지우기"

#: ../DistUpgrade/DistUpgradeCache.py:133
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"'%s' 패키지는 불완전한 상태여서 재설치가 필요하지만, 아카이브에서 찾을 수 없습니다. 이 패키지를 지우고 계속하길 원하십니까?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:231
msgid "The server may be overloaded"
msgstr "서버가 과부화 상태인 것 같습니다"

#: ../DistUpgrade/DistUpgradeCache.py:341
msgid "Broken packages"
msgstr "망가진 패키지"

#: ../DistUpgrade/DistUpgradeCache.py:342
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr "이 소프트웨어로 고칠 수 없는 망가진 패키지가 있습니다. 진행하기 전에 시냅틱이나 apt-get을 사용하여 복구하십시오."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:631
#, python-format
msgid ""
"An unresolvable problem occurred while calculating the upgrade:\n"
"%s\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"업데이트를 계산하는 중 해결할 수 없는 문제가 생겼습니다.:\n"
"%s\n"
"\n"
" 이것의 원인은 다음 중 하나일 수 있습니다:\n"
" * 차기 버전의 우분투 업그레이드로 업그레이드 중이거나\n"
" * 차기 버전의 우분투를 사용하고 있거나\n"
" * 써드 파티 소프트웨어를 사용 중입니다.\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:641
msgid "This is most likely a transient problem, please try again later."
msgstr "이것은 아마도 일시적인 문제일 것입니다. 나중에 다시 시도해 주시기 바랍니다."

#: ../DistUpgrade/DistUpgradeCache.py:644
msgid ""
"If none of this applies, then please report this bug against the 'update-"
"manager' package and include the files in /var/log/dist-upgrade/ in the bug "
"report."
msgstr ""
"만약 적용되지 않는다면, /var/log/dist-upgrade/ 에 있는 파일들을 버그 보고서에 넣어서 'update-manager' "
"패키지에 이 버그를 보고해 주시기 바랍니다."

#: ../DistUpgrade/DistUpgradeCache.py:650
#: ../UpdateManager/UpdateManager.py:870
msgid "Could not calculate the upgrade"
msgstr "업그레이드에 필요한 의존성을 계산할 수 없습니다."

#: ../DistUpgrade/DistUpgradeCache.py:700
msgid "Error authenticating some packages"
msgstr "패키지 인증 오류"

#: ../DistUpgrade/DistUpgradeCache.py:701
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"인증할 수 없는 패키지가 있습니다. 일시적인 네트워크 문제일 수 있으니 이 경우라면 나중에 다시 시도하십시오. 인증되지 않은 패키지의 "
"목록은 다음과 같습니다."

#: ../DistUpgrade/DistUpgradeCache.py:721
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr "이 패키지 '%s'은(는) 제거 블랙리스트에 적혀 있어서 제거할 수 없습니다."

#: ../DistUpgrade/DistUpgradeCache.py:725
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "필수 패키지 '%s' 가 삭제로 표시 되었습니다."

#: ../DistUpgrade/DistUpgradeCache.py:734
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "블랙리스트에 있는 버전 '%s'을(를) 설치하려고 함"

#: ../DistUpgrade/DistUpgradeCache.py:836
#, python-format
msgid "Can not mark '%s' for upgrade"
msgstr "'%s'을(를) 업데이트 항목으로 표시할 수 없습니다."

#: ../DistUpgrade/DistUpgradeCache.py:850
#, python-format
msgid "Can't install '%s'"
msgstr "'%s'을(를) 설치할 수 없습니다."

#: ../DistUpgrade/DistUpgradeCache.py:851
msgid ""
"It was impossible to install a required package. Please report this as a "
"bug. "
msgstr "요청된 패키지를 설치할 수 없습니다. 버그로 보고하여 주십시오. "

#. FIXME: provide a list
#: ../DistUpgrade/DistUpgradeCache.py:860
msgid "Can't guess meta-package"
msgstr "메타 패키지를 추측할 수 없습니다."

#: ../DistUpgrade/DistUpgradeCache.py:861
msgid ""
"Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
"desktop or edubuntu-desktop package and it was not possible to detect which "
"version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"시스템에 ubuntu-desktop이나 kubuntu-desktop이나 xubuntu-desktop 또는 edubuntu-desktop "
"패키지가 없으며, 현재 실행중인 우분투의 버전을 알아낼 수 없습니다.\n"
" 위의 패키지 중 하나를 시냅틱이나 apt-get을 이용해서 먼저 설치하시기 바랍니다."

#: ../DistUpgrade/DistUpgradeController.py:86
msgid "Reading cache"
msgstr "캐시를 읽는 중"

#: ../DistUpgrade/DistUpgradeController.py:189
msgid "Unable to get exclusive lock"
msgstr "배타적인 잠금을 할 수 없습니다"

#: ../DistUpgrade/DistUpgradeController.py:190
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"이것은 보통 또 다른 패키지 관리 프로그램(예를 들어 apt-get이나 aptitude)이 이미 실행 중임을 의미 합니다. 우선 그 "
"프로그램을 종료 하십시오."

#: ../DistUpgrade/DistUpgradeController.py:236
msgid "Upgrading over remote connection not supported"
msgstr "원격 접속을 통한 업그레이드는 지원 안 됨"

#: ../DistUpgrade/DistUpgradeController.py:237
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"당신은 원격 SSH접속을 통한 업그레이드를 지원하지 않는 SSH프론트엔드를 사용하여 업그레이드를 실행하고 있습니다. 'do-release-"
"upgrade' 명령을 이용하여 텍스트모드 업그레이드를 시도하십시오.\n"
"업그레이드가 중지될 것입니다. SSH를 사용하지 않고 다시 시도하십시오."

#: ../DistUpgrade/DistUpgradeController.py:251
msgid "Continue running under SSH?"
msgstr "SSH를 통해 실행하는 것을 계속 진행하시겠습니까?"

#: ../DistUpgrade/DistUpgradeController.py:252
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"현재 세션은 SSH에서 돌아가고 있습니다. SSH를 이용한 세션에서 업그레이드를 하는 것은 추천하지 않습니다. 업그레이드에 실패할 경우 "
"복구하기가 힘들어 집니다.\n"
"진행하시면 '%s' 포트로 추가적인 SSH데몬이 시작될 것입니다.\n"
"계속하시겠습니까?"

#: ../DistUpgrade/DistUpgradeController.py:267
msgid "Starting additional sshd"
msgstr "추가적인 sshd 시작하기"

#: ../DistUpgrade/DistUpgradeController.py:268
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"문제가 발생했을 때 복구를 쉽게 할 수 있도록 sshd 데몬을 포트 '%s'에 추가로 실행합니다. 현재 실행중인 ssh에 문제가 생겨도 "
"추가로 실행한 데몬으로 접속할 수 있습니다.\n"

#: ../DistUpgrade/DistUpgradeController.py:339
#: ../DistUpgrade/DistUpgradeController.py:384
msgid "Can not upgrade"
msgstr "업그레이드할 수 없습니다"

#: ../DistUpgrade/DistUpgradeController.py:340
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "이 도구를 통해 '%s'에서 '%s'(으)로 업그레이드하는 것은 지원되지 않습니다."

#: ../DistUpgrade/DistUpgradeController.py:349
msgid "Sandbox setup failed"
msgstr "샌드박스 장치 조정 실패"

#: ../DistUpgrade/DistUpgradeController.py:350
msgid "It was not possible to create the sandbox environment."
msgstr "샌드박스 환경 생성이 불가능 했습니다."

#: ../DistUpgrade/DistUpgradeController.py:356
msgid "Sandbox mode"
msgstr "샌드박스 모드"

#: ../DistUpgrade/DistUpgradeController.py:357
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a systemdir from now until the next reboot are "
"permanent."
msgstr ""
"이 업그레이드는 샌드박스 (시험) 모드에서 실행되고 있습니다.  모든 변경사항들은 '%s'에 씌여지며 다음 다시 시작 때 지워질 것 "
"입니다.\n"
"\n"
"지금부터 다음 다시 시작 때까지 시스템 디렉터리 내용은 전혀 변경되지 *않습니다.*"

#: ../DistUpgrade/DistUpgradeController.py:385
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr "파이썬 설치가 잘못 되었습니다. '/usr/bin/python' 심볼릭 링크를 수정하십시오."

#: ../DistUpgrade/DistUpgradeController.py:411
msgid "Package 'debsig-verify' is installed"
msgstr "'debsig-verify' 꾸러미가 설치되어 있습니다."

#: ../DistUpgrade/DistUpgradeController.py:412
msgid ""
"The upgrade can not continue with that package installed.\n"
"Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
"run the upgrade again."
msgstr ""
"그 패키지가 설치된 상태로는 업그레이드를 계속할 수 없습니다.\n"
"시냅틱이나 'apt-get remove debsig-verify' 명령으로 그것을 제거한 후 다시 업그레이드를 수행하십시오."

#: ../DistUpgrade/DistUpgradeController.py:422
msgid "Include latest updates from the Internet?"
msgstr "인터넷에서 최신 업데이트를 설치하겠습니까?"

#: ../DistUpgrade/DistUpgradeController.py:423
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"업그레이드 프로그램은 업그레이드 과정에서 인터넷을 이용해 자동으로 최신 업데이트를 내려받아 설치할 수 있습니다. 인터넷에 연결되어 있다면 "
"가급적 설치하는 것이 좋습니다.\n"
"\n"
"업그레이드는 더 오래 걸리겠지만 완료 후에는 시스템이 완전히 최신 상태가 됩니다. 지금 설치를 하지 않기로 했다면 업그레이드를 마친 다음 "
"즉시 최신 업데이트를 설치해야 합니다.\n"
"'아니오'를 선택하면 네트워크를 이용하지 않습니다."

#: ../DistUpgrade/DistUpgradeController.py:639
#, python-format
msgid "disabled on upgrade to %s"
msgstr "%s로 업그레이드 하는 것이 비활성화 됨"

#: ../DistUpgrade/DistUpgradeController.py:666
msgid "No valid mirror found"
msgstr "적합한 미러 서버를 찾지 못했습니다."

#: ../DistUpgrade/DistUpgradeController.py:667
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run a internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"저장소 정보를 찾아봤지만 업그레이드를 위한 미러 항목을 찾을 수 없습니다. 내부 미러를 운영중이거나 혹은 미러 정보가 최신이 아닐 수 "
"있습니다.\n"
"\n"
"그래도 'sources.list' 파일을 재작성 하시겠습니까? 여기서 '예'를 선택하면 모든 '%s' 항목은 '%s'(으)로  업데이트 "
"됩니다.\n"
"'아니오'를 선택하면 업그레이드는 취소됩니다."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:687
msgid "Generate default sources?"
msgstr "기본 설정된 소스 목록을 만드시겠습니까?"

#: ../DistUpgrade/DistUpgradeController.py:688
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"'sources.list' 파일을 찾아봤지만 '%s'에 대한 항목을 찾을 수 없습니다.\n"
"'%s'에 대한 기본 항목을 추가하시겠습니까? '아니오'를 선택하면 업그레이드는 취소됩니다."

#: ../DistUpgrade/DistUpgradeController.py:723
msgid "Repository information invalid"
msgstr "저장소 정보가 올바르지 않습니다"

#: ../DistUpgrade/DistUpgradeController.py:724
msgid ""
"Upgrading the repository information resulted in a invalid file. Please "
"report this as a bug."
msgstr "저장소 정보를 업그레이드했는데 잘못된 파일이 만들어졌습니다. 버그를 보고하여 주십시오."

#: ../DistUpgrade/DistUpgradeController.py:730
msgid "Third party sources disabled"
msgstr "써드 파티 소스는 이용할 수 없습니다"

#: ../DistUpgrade/DistUpgradeController.py:731
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"souces.list에서 써드 파티 목록의 일부는 이용할 수 없게 되었습니다. 업그레드를 마친 후 '소프트웨어 소스' 도구나 꾸러미 "
"관리자를 이용해 다시 사용가능하게 할 수 있습니다."

#: ../DistUpgrade/DistUpgradeController.py:771
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "패키지 상태 불일치"

#: ../DistUpgrade/DistUpgradeController.py:774
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"'%s' 패키지들은 불완전한 상태이며 재설치될 필요가 있습니다만, 아카이브에서 찾을 수가 없습니다.  이 패키지들을 수동으로 재설치하거나 "
"시스템에서 제거하시기 바랍니다."

#: ../DistUpgrade/DistUpgradeController.py:822
msgid "Error during update"
msgstr "업데이트 중 오류"

#: ../DistUpgrade/DistUpgradeController.py:823
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"업데이트를 하는 동안에 문제가 발생했습니다. 보통 네트워크 문제인 경우가 많습니다. 네트워크의 연결 상태를 확인하시고 다시 시도하십시오."

#. print "on_button_install_clicked"
#: ../DistUpgrade/DistUpgradeController.py:832
#: ../UpdateManager/UpdateManager.py:628
msgid "Not enough free disk space"
msgstr "디스크 여유 공간이 충분하지 않습니다."

#: ../DistUpgrade/DistUpgradeController.py:833
#, python-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of %s free space on disk "
"'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
"your trash and remove temporary packages of former installations using 'sudo "
"apt-get clean'."
msgstr ""
"업그레이드가 중단되었습니다. 이 업그레이드는 %s 의 용량이 다음 드라이브 '%s'에 필요합니다. 최소한 %s의 용량을 '%s'에 "
"확보해주세요. 휴지통을 비워주시고 'sudo apt-get clean' 명령어를 통해 이전 설치에서 생긴 임시 패키지를 지워주세요."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:861
#: ../DistUpgrade/DistUpgradeController.py:1577
msgid "Calculating the changes"
msgstr "변경 사항을 적용하는 중"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:893
msgid "Do you want to start the upgrade?"
msgstr "업그레이드를 시작하시겠습니까?"

#: ../DistUpgrade/DistUpgradeController.py:959
msgid "Upgrade canceled"
msgstr "업그레이드가 취소되었습니다."

#: ../DistUpgrade/DistUpgradeController.py:960
msgid ""
"The upgrade cancels now and the original system state is restored. You can "
"resume the upgrade at a later time."
msgstr "현재 업그레이드가 취소되어 원래 시스템 상태로 복원되었습니다. 나중에 업그레이드를 다시 시작할 수 있습니다."

#: ../DistUpgrade/DistUpgradeController.py:965
#: ../DistUpgrade/DistUpgradeController.py:1064
msgid "Could not download the upgrades"
msgstr "업그레이드 파일을 다운로드 할 수 없습니다."

#: ../DistUpgrade/DistUpgradeController.py:966
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far are kept."
msgstr ""
"업그레이드가 중단되었습니다. 인터넷 연결이나 설치 매체를 확인해보시고 다시 시도해보세요. 이미 다운된 파일들은 보관 중입니다."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1016
#: ../DistUpgrade/DistUpgradeController.py:1052
#: ../DistUpgrade/DistUpgradeController.py:1157
msgid "Error during commit"
msgstr "커밋 도중 오류 발생"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1018
#: ../DistUpgrade/DistUpgradeController.py:1054
#: ../DistUpgrade/DistUpgradeController.py:1196
msgid "Restoring original system state"
msgstr "시스템을 원래의 상태로 복구하고 있습니다"

#: ../DistUpgrade/DistUpgradeController.py:1019
#: ../DistUpgrade/DistUpgradeController.py:1033
#: ../DistUpgrade/DistUpgradeController.py:1055
msgid "Could not install the upgrades"
msgstr "업그레이드를 설치하지 못했습니다."

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1024
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"업그레이드가 중단되었습니다. 시스템에 중대한 오류가 발생했을 가능성이 있습니다. 복구(dpkg --configure -a)를 "
"실행하겠습니다."

#: ../DistUpgrade/DistUpgradeController.py:1029
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"files in /var/log/dist-upgrade/ in the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"이 버그를 'update-manager' 패키지에 보고해 주시고, /var/log/dist-upgrade/에 있는 파일들을 버그 보고서에 "
"첨부해 주세요.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1065
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr "업그레이드가 중단되었습니다. 인터넷 연결이나 설치 매체를 확인해보시고 다시 시도해보세요. "

#: ../DistUpgrade/DistUpgradeController.py:1145
msgid "Remove obsolete packages?"
msgstr "못쓰게 된 꾸러미를 삭제하시겠습니까?"

#: ../DistUpgrade/DistUpgradeController.py:1146
#: ../DistUpgrade/DistUpgrade.ui.h:22
msgid "_Keep"
msgstr "유지(_K)"

#: ../DistUpgrade/DistUpgradeController.py:1146
msgid "_Remove"
msgstr "삭제(_R)"

#: ../DistUpgrade/DistUpgradeController.py:1158
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr "정리하는 도중에 문제가 발생하였습니다. 다음의 메시지에서 더 많은 정보를 확인할 수 있습니다. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1234
msgid "Required depends is not installed"
msgstr "필요한 의존 프로그램이 설치되어 있지 않습니다."

#: ../DistUpgrade/DistUpgradeController.py:1235
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "필요한 의존 프로그램 '%s' (이)가 설치되어 있지 않습니다. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1480
#: ../DistUpgrade/DistUpgradeController.py:1547
msgid "Checking package manager"
msgstr "꾸러미 관리자 확인 중"

#: ../DistUpgrade/DistUpgradeController.py:1485
msgid "Preparing the upgrade failed"
msgstr "업그레이드 준비에 실패했습니다"

#: ../DistUpgrade/DistUpgradeController.py:1486
msgid ""
"Preparing the system for the upgrade failed. Please report this as a bug "
"against the 'update-manager' package and include the files in /var/log/dist-"
"upgrade/ in the bug report."
msgstr ""
"업그레이드를 위한 시스템 준비 작업에 실패했습니다. 이 버그를 'update-manager' 패키지에 보고해 주시고, "
"/var/log/dist-upgrade/에 있는 파일들을 버그 보고서에 첨부해 주세요."

#: ../DistUpgrade/DistUpgradeController.py:1502
msgid "Getting upgrade prerequisites failed"
msgstr "업그레이드 준비에 실패했습니다"

#: ../DistUpgrade/DistUpgradeController.py:1503
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Please report this as a bug against the 'update-manager' package and include "
"the files in /var/log/dist-upgrade/ in the bug report."
msgstr ""
"시스템이 업그레이드를 위한 선행조건들을 가져올 수 없었습니다.  업그레이드는 이제 중단되며 시스템은 기존 상태로 복원될 것입니다.\n"
"\n"
"이 상황을 버그로 'update-manager' 패키지에 보고해 주시고, /var/log/dist-upgrade/에 있는 파일들을 버그 "
"보고서에 첨부해 주세요."

#: ../DistUpgrade/DistUpgradeController.py:1533
msgid "Updating repository information"
msgstr "저장소 정보 업데이트 중"

#: ../DistUpgrade/DistUpgradeController.py:1565
msgid "Invalid package information"
msgstr "잘못된 꾸러미 정보"

#: ../DistUpgrade/DistUpgradeController.py:1566
#, python-format
msgid ""
"After your package information was updated the essential package '%s' can "
"not be found anymore.\n"
"This indicates a serious error, please report this bug against the 'update-"
"manager' package and include the files in /var/log/dist-upgrade/ in the bug "
"report."
msgstr ""
"당신의 패키지 정보가 업데이트 된 후, 필수 패키지 '%s'를 어디에서도 찾을 수가 없습니다.\n"
"이는 심각한 오류이며, 이 버그를 'update-manager' 패키지에 보고해 주시고, /var/log/dist-upgrade/에 있는 "
"파일들을 버그 보고서에 첨부해 주세요."

#: ../DistUpgrade/DistUpgradeController.py:1583
#: ../DistUpgrade/DistUpgradeController.py:1631
msgid "Fetching"
msgstr "가져오는 중"

#: ../DistUpgrade/DistUpgradeController.py:1589
#: ../DistUpgrade/DistUpgradeController.py:1635
msgid "Upgrading"
msgstr "업그레이드 중"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1594
#: ../DistUpgrade/DistUpgradeController.py:1637
#: ../DistUpgrade/DistUpgradeController.py:1644
#: ../DistUpgrade/DistUpgradeController.py:1655
msgid "Upgrade complete"
msgstr "업그레이드 완료"

#: ../DistUpgrade/DistUpgradeController.py:1595
#: ../DistUpgrade/DistUpgradeController.py:1638
#: ../DistUpgrade/DistUpgradeController.py:1645
msgid ""
"The upgrade is completed but there were errors during the upgrade process."
msgstr "업그레이드 진행 중 오류가 있었으나 완료되었습니다."

#: ../DistUpgrade/DistUpgradeController.py:1602
msgid "Searching for obsolete software"
msgstr "못 쓰게 된 소프트웨어를 검색하는 중"

#: ../DistUpgrade/DistUpgradeController.py:1607
msgid "System upgrade is complete."
msgstr "시스템 업그레이드가 완료되었습니다."

#: ../DistUpgrade/DistUpgradeController.py:1656
msgid "The partial upgrade was completed."
msgstr "부분 업그레이드가 완료되었습니다."

#: ../DistUpgrade/DistUpgradeQuirks.py:168
msgid "evms in use"
msgstr "evms 사용"

#: ../DistUpgrade/DistUpgradeQuirks.py:169
msgid ""
"Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
"software is no longer supported, please switch it off and run the upgrade "
"again when this is done."
msgstr ""
"이 시스템은 /proc/mounts 안에 'evms' 볼륨 매니저를 사용하고 있습니다.  'evms'은 더이상 지원되지 않으므로, 종료 "
"후에 업그레이드를 다시 실행하십시오."

#: ../DistUpgrade/DistUpgradeQuirks.py:434
#: ../DistUpgrade/DistUpgradeQuirks.py:462
#: ../DistUpgrade/DistUpgradeQuirks.py:489
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr "업그레이드로 인해 데스크탑 효과나 게임, 그래픽 관련 프로그램의 성능이 줄어들 수 있습니다."

#: ../DistUpgrade/DistUpgradeQuirks.py:438
#: ../DistUpgrade/DistUpgradeQuirks.py:466
msgid ""
"This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
"version of this driver is available that works with your video card in "
"Ubuntu 10.04 LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"이 컴퓨터는 현재 NVIDIA사의 'nvidia' 그래픽 드라이버를 사용중입니다. 이 드라이버는 우분투 10.04 LTS에 대응되는 "
"버전이 없습니다.\n"
"진행하시겠습니까?"

#: ../DistUpgrade/DistUpgradeQuirks.py:493
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"이 컴퓨터는 현재 AMD사의 'fglrx' 그래픽 드라이버를 사용중입니다. 이 드라이버는 우분투 10.04 LTS에 대응되는 버전이 "
"없습니다.\n"
"진행하시겠습니까?"

#: ../DistUpgrade/DistUpgradeQuirks.py:520
msgid "No ARMv6 CPU"
msgstr "ARMv6 CPU가 아님"

#: ../DistUpgrade/DistUpgradeQuirks.py:521
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"시스템이 ARMv6 보다 더 오래된 아키텍쳐의 ARM CPU를 사용하고 있습니다. 우분투 칼믹 코알라의 모든 패키지는 ARMv6에 최적화 "
"되어있기 때문에 최소 CPU로 ARMv6을 필요로 합니다. 현재 하드웨어로는 새로운 우분투 버전으로 시스템을 업그레이드 할 수 없습니다."

#: ../DistUpgrade/DistUpgradeQuirks.py:541
msgid "No init available"
msgstr "init 가능하지 않습니다"

#: ../DistUpgrade/DistUpgradeQuirks.py:542
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"현재 시스템은 init 데몬(예:Linux-VServer)가 없는 가상환경으로 감지되었습니다. 우분투 10.04 LTS는 이러한 환경에서 "
"제대로 동작할 수 없으며 가상 환경의 설정에 조정이 필요합니다.\n"
"계속 진행하시겠습니까?"

#: ../DistUpgrade/DistUpgradeMain.py:47
msgid "Sandbox upgrade using aufs"
msgstr "aufs를 이용하여 샌드박스 업그레이드를 합니다."

#: ../DistUpgrade/DistUpgradeMain.py:49
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr "주어진 경로를 이용하여 업그레이드할 수 있는 패키지가 있는 시디롬을 찾습니다."

#: ../DistUpgrade/DistUpgradeMain.py:55
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"프론트 엔드를 사용합니다. 현재 이용가능한 것들은 다음과 같습니다: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:58
msgid "*DEPRECATED* this option will be ignore"
msgstr "*삭제된 옵션* 이 항목은 무시됩니다."

#: ../DistUpgrade/DistUpgradeMain.py:61
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr "부분 업그레이드만 실행합니다.(sources.list 는 변경하지 않음)"

#: ../DistUpgrade/DistUpgradeMain.py:63
msgid "Set datadir"
msgstr "datadir 설정"

#. print "mediaChange %s %s" % (medium, drive)
#: ../DistUpgrade/DistUpgradeViewGtk.py:113
#: ../DistUpgrade/DistUpgradeViewKDE.py:190
#: ../UpdateManager/DistUpgradeFetcherKDE.py:148
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "'%s'(을)를 '%s' 드라이브에 넣으십시오."

#: ../DistUpgrade/DistUpgradeViewGtk.py:134
#: ../DistUpgrade/DistUpgradeViewKDE.py:204
msgid "Fetching is complete"
msgstr "완전히 받았습니다"

#: ../DistUpgrade/DistUpgradeViewGtk.py:145
#: ../DistUpgrade/DistUpgradeViewKDE.py:217
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "%li번째 파일 (전체 %li개) %sB/s로 내려받는 중"

#: ../DistUpgrade/DistUpgradeViewGtk.py:148
#: ../DistUpgrade/DistUpgradeViewGtk.py:288
#: ../DistUpgrade/DistUpgradeViewKDE.py:218
#: ../DistUpgrade/DistUpgradeViewKDE.py:358
#, python-format
msgid "About %s remaining"
msgstr "%s 정도 남음"

#: ../DistUpgrade/DistUpgradeViewGtk.py:151
#: ../DistUpgrade/DistUpgradeViewKDE.py:220
#, python-format
msgid "Fetching file %li of %li"
msgstr "%li번째 파일 (전체 %li개) 내려받는 중"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk.py:183
#: ../DistUpgrade/DistUpgradeViewKDE.py:254
msgid "Applying changes"
msgstr "변경 사항을 적용하는 중"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk.py:208
#: ../DistUpgrade/DistUpgradeViewKDE.py:267
msgid "dependency problems - leaving unconfigured"
msgstr "의존성 문제 - 설정하지 않고 남겨둠"

#: ../DistUpgrade/DistUpgradeViewGtk.py:213
#: ../DistUpgrade/DistUpgradeViewKDE.py:269
#, python-format
msgid "Could not install '%s'"
msgstr "'%s'(을)를 설치할 수 없습니다."

#: ../DistUpgrade/DistUpgradeViewGtk.py:214
#: ../DistUpgrade/DistUpgradeViewKDE.py:270
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr "업그레이드는 계속 진행되겠지만 '%s' 패키지가 동작하지 않을 수도 있습니다. 이에 대한 버그 리포트를 보내주십시오."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk.py:231
#: ../DistUpgrade/DistUpgradeViewKDE.py:291
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"바뀐 설정 파일 '%s'을(를)\n"
"교체하시겠습니까?"

#: ../DistUpgrade/DistUpgradeViewGtk.py:232
#: ../DistUpgrade/DistUpgradeViewKDE.py:292
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr "새로운 버전으로 대체하기를 선택하면 이전 설정 파일에서 변경한 내용들을 잃게 됩니다."

#: ../DistUpgrade/DistUpgradeViewGtk.py:246
#: ../DistUpgrade/DistUpgradeViewKDE.py:310
msgid "The 'diff' command was not found"
msgstr "'diff' 명령어를 찾지 못했습니다."

#: ../DistUpgrade/DistUpgradeViewGtk.py:454
#: ../DistUpgrade/DistUpgradeViewText.py:88
msgid "A fatal error occurred"
msgstr "심각한 오류 발생"

#: ../DistUpgrade/DistUpgradeViewGtk.py:455
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"이를 버그로 신고해주시고 버그 보고서에 /var/log/dist-upgrade/main.log와 /var/log/dist-"
"upgrade/apt.log를 첨부해주세요. 업그레이드가 중단되었습니다.\n"
"기존 sources.list 파일은 /etc/apt/sources.list.distUpgrade에 저장되었습니다."

#: ../DistUpgrade/DistUpgradeViewGtk.py:472
msgid "Ctrl-c pressed"
msgstr "Ctrl-c 눌림"

#: ../DistUpgrade/DistUpgradeViewGtk.py:473
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr "이렇게 작업을 취소하면 시스템이 깨진 상태가 됩니다. 정말 그렇게 할까요?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk.py:621
msgid "To prevent data loss close all open applications and documents."
msgstr "데이터 손실을 막으려면 열려있는 모든 프로그램과 문서를 닫으십시오."

#: ../DistUpgrade/DistUpgradeViewGtk.py:635
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "캐노니컬에서 더이상 지원되지 않습니다 (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:636
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>다운그레이드 (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk.py:637
#, python-format
msgid "Remove (%s)"
msgstr "제거 (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:638
#, python-format
msgid "No longer needed (%s)"
msgstr "더 이상 필요하지 않음 (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:639
#, python-format
msgid "Install (%s)"
msgstr "설치 (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk.py:640
#, python-format
msgid "Upgrade (%s)"
msgstr "업그레이드(%s)"

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../DistUpgrade/DistUpgradeViewKDE.py:191
#: ../UpdateManager/DistUpgradeFetcherKDE.py:150
msgid "Media Change"
msgstr "미디어 변경"

#: ../DistUpgrade/DistUpgradeViewKDE.py:322
msgid "Show Difference >>>"
msgstr "차이점 보기 >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:325
msgid "<<< Hide Difference"
msgstr "<<< 차이점 숨기기"

#: ../DistUpgrade/DistUpgradeViewKDE.py:539
msgid "Error"
msgstr "오류"

#: ../DistUpgrade/DistUpgradeViewKDE.py:551
msgid "&Cancel"
msgstr "취소(&C)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:553
#: ../DistUpgrade/DistUpgradeViewKDE.py:795
msgid "&Close"
msgstr "닫기(&C)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:599
msgid "Show Terminal >>>"
msgstr "터미널 보이기 >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:602
msgid "<<< Hide Terminal"
msgstr "<<< 터미널 숨기기"

#: ../DistUpgrade/DistUpgradeViewKDE.py:682
msgid "Information"
msgstr "정보"

#: ../DistUpgrade/DistUpgradeViewKDE.py:733
#: ../DistUpgrade/DistUpgradeViewKDE.py:778
#: ../DistUpgrade/DistUpgradeViewKDE.py:781 ../DistUpgrade/DistUpgrade.ui.h:11
msgid "Details"
msgstr "자세히"

#: ../DistUpgrade/DistUpgradeViewKDE.py:759
#, python-format
msgid "No longer supported %s"
msgstr "지원하지 않음 %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:761
#, python-format
msgid "Remove %s"
msgstr "%s 제거"

#: ../DistUpgrade/DistUpgradeViewKDE.py:763
#: ../DistUpgrade/DistUpgradeViewText.py:175
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "%s 제거 (자동설치된)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:765
#, python-format
msgid "Install %s"
msgstr "%s 설치"

#: ../DistUpgrade/DistUpgradeViewKDE.py:767
#, python-format
msgid "Upgrade %s"
msgstr "%s 업그레이드"

#: ../DistUpgrade/DistUpgradeViewKDE.py:791
#: ../DistUpgrade/DistUpgradeViewText.py:223
msgid "Restart required"
msgstr "재시작이 필요합니다."

#: ../DistUpgrade/DistUpgradeViewKDE.py:791
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>업그레이드가 완료되어 시스템을 재시작합니다.</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:794 ../DistUpgrade/DistUpgrade.ui.h:25
#: ../data/glade/UpdateManager.ui.h:31
msgid "_Restart Now"
msgstr "지금 바로 재시작(_R)"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:812
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>진행중인 업그레이드를 취소할까요?</big></b>\n"
"\n"
"업그레이드를 도중에 취소하면 시스템이 사용 불가능한 상태가 될 수도 있습니다. 업그레이드를 계속 진행하는 것이 좋습니다."

#: ../DistUpgrade/DistUpgradeViewKDE.py:816
msgid "Cancel Upgrade?"
msgstr "업그레이드를 취소하겠습니까?"

#: ../DistUpgrade/DistUpgradeView.py:60
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li 일"

#: ../DistUpgrade/DistUpgradeView.py:62
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li 시간"

#: ../DistUpgrade/DistUpgradeView.py:64
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li 분"

#: ../DistUpgrade/DistUpgradeView.py:65
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li 초"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:81
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:99
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:147
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr "이 다운로드는 1Mbit DSL 연결로는 약 %s, 56k 모뎀으로는 약 %s 정도 걸릴 것입니다."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:151
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "현재 연결 상태로 다운로드가 약 %s 정도 걸릴 것입니다. "

#: ../DistUpgrade/DistUpgradeView.py:253 ../DistUpgrade/DistUpgrade.ui.h:16
msgid "Preparing to upgrade"
msgstr "업그레이드를 준비하는 중"

#: ../DistUpgrade/DistUpgradeView.py:254
msgid "Getting new software channels"
msgstr "소프트웨어 채널을 가져오는 중"

#: ../DistUpgrade/DistUpgradeView.py:255 ../DistUpgrade/DistUpgrade.ui.h:14
msgid "Getting new packages"
msgstr "패키지를 가져오는 중"

#: ../DistUpgrade/DistUpgradeView.py:256 ../DistUpgrade/DistUpgrade.ui.h:15
msgid "Installing the upgrades"
msgstr "업그레이드를 설치하는 중"

#: ../DistUpgrade/DistUpgradeView.py:257 ../DistUpgrade/DistUpgrade.ui.h:10
msgid "Cleaning up"
msgstr "지우는 중"

#: ../DistUpgrade/DistUpgradeView.py:341
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"설치된 패키지 중 %(amount)d 개는 더 이상 캐노니컬이 지원하지 않습니다. 하지만 공동체로부터의 지원은 계속됩니다."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:350
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "패키지 %d개를 삭제할 것입니다."

#: ../DistUpgrade/DistUpgradeView.py:355
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "새로운 패키지 %d개를 설치할 것입니다."

#: ../DistUpgrade/DistUpgradeView.py:361
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "패키지 %d개를 업그레이드할 것입니다."

#: ../DistUpgrade/DistUpgradeView.py:366
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"총 %s개의 패키지를 다운로드해야 합니다. "

#: ../DistUpgrade/DistUpgradeView.py:370
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be cancelled."
msgstr "업그레이드를 다운로드 하여 설치하는 것은 여러 시간이 걸릴 수도 있으며, 일단 다운로드가 완료되면 취소할 수 없습니다."

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:376 ../UpdateManager/UpdateManager.py:557
msgid "Your system is up-to-date"
msgstr "시스템이 최신의 상태입니다."

#: ../DistUpgrade/DistUpgradeView.py:377
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr "시스템에 업그레이드할 것이 없습니다. 업그레이드를 취소합니다."

#: ../DistUpgrade/DistUpgradeView.py:390
msgid "Reboot required"
msgstr "재부팅이 필요합니다."

#: ../DistUpgrade/DistUpgradeView.py:391
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr "업그레이드가 끝났으며 다시 시작해야 합니다. 지금 하시겠습니까?"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:135
#: ../DistUpgrade/DistUpgradeFetcherCore.py:136
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:135
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:136
msgid "Could not run the upgrade tool"
msgstr "업그레이드 도구를 실행할 수 없습니다."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:136
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:136
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug"
msgstr "업그레이드 도구에 버그가 있을 가능성이 높습니다. 버그를 보고하여 주십시오."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:205
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:205
msgid "Upgrade tool signature"
msgstr "업그레이드 도구 서명"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:208
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:208
msgid "Upgrade tool"
msgstr "업그레이드 도구"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:237
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:237
msgid "Failed to fetch"
msgstr "가져오기에 실패했습니다."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:238
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:238
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr "업그레이드를 가져오는 데 실패했습니다. 네트워크에 문제가 있을 수 있습니다. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:242
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:242
msgid "Failed to extract"
msgstr "압축 풀기 실패"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:243
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr "업그레이드의 압축을 푸는 데 실패했습니다. 네트워크나 서버에 문제가 있을 수 있습니다. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:248
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:248
msgid "Verification failed"
msgstr "검증 실패"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:249
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr "업그레이드를 확인하는 데 실패했습니다.  네트워크나 서버에 문제가 있을 수 있습니다. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:254
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:254
msgid "Authentication failed"
msgstr "인증 실패"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:255
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:255
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr "업그레이드를 인증하는 데 실패했습니다.네트워크나 서버에 문제가 있을 수 있습니다. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:269
#: ../DistUpgrade/DistUpgradeFetcherCore.py:275
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:269
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:275
msgid "Can not run the upgrade"
msgstr "업그레이드를 할 수 없습니다."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:270
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:270
msgid ""
"This usually is caused by a system were /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"이는 보통 /tmp 디렉토리가 noexec로 마운트된 시스템에서 발생합니다. noexec를 마운트 해제하고 업그레이드를 다시 시작하십시오."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:276
#: ../UpdateManager/Core/DistUpgradeFetcherCore.py:276
#, python-format
msgid "The error message is '%s'."
msgstr "오류 메시지 '%s'"

#: ../DistUpgrade/DistUpgradeViewText.py:89
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"/var/log/dist-upgrade/main.log와 /var/log/dist-upgrade/apt.log를 보고서에 첨부하여 버그 "
"보고를 해주십시오. 업그레이드를 취소합니다.\n"
"원본 소스 목록을 /etc/apt/sources.list.distUpgrade에 저장하였습니다."

#: ../DistUpgrade/DistUpgradeViewText.py:113
msgid "Aborting"
msgstr "중지하는 중"

#: ../DistUpgrade/DistUpgradeViewText.py:118
msgid "Demoted:\n"
msgstr "강등됨:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:151
#: ../DistUpgrade/DistUpgradeViewText.py:189
#: ../DistUpgrade/DistUpgradeViewText.py:196
msgid "Continue [yN] "
msgstr "계속 [yN] "

#: ../DistUpgrade/DistUpgradeViewText.py:151
#: ../DistUpgrade/DistUpgradeViewText.py:189
msgid "Details [d]"
msgstr "상세 정보 [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:155
#: ../DistUpgrade/DistUpgradeViewText.py:199
msgid "y"
msgstr "y"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:158
#: ../DistUpgrade/DistUpgradeViewText.py:206
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:161
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:166
#, python-format
msgid "No longer supported: %s\n"
msgstr "더이상 지원되지 않음: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:171
#, python-format
msgid "Remove: %s\n"
msgstr "제거: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:181
#, python-format
msgid "Install: %s\n"
msgstr "설지: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:186
#, python-format
msgid "Upgrade: %s\n"
msgstr "업그레이드: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:203
msgid "Continue [Yn] "
msgstr "계속하겠습니까? [Yn] "

#: ../DistUpgrade/DistUpgradeViewText.py:224
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"업그레이드를 완료하시려면 재시작이 필요합니다.\n"
"'y'를 선택하시면 시스템이 재시작 됩니다."

#: ../DistUpgrade/DistUpgrade.ui.h:1
msgid "    "
msgstr "    "

#: ../DistUpgrade/DistUpgrade.ui.h:2
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>실행 중인 업그레이드를 취소하시겠습니까?</big></b>\n"
"\n"
"업그레이드를 취소하면 시스템이 사용하기 어려운 상태가 될 수 있습니다. 업그레이드를 계속 하시는 쪽을 강력히 추천합니다."

#: ../DistUpgrade/DistUpgrade.ui.h:5
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr "<b><big>시스탬을 재시작 하면 성공적으로 업데이트가 됩니다.</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:8
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>업그레이드를 시작하시겠습니까?</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:9
msgid "<b><big>Upgrading Ubuntu to version 10.10</big></b>"
msgstr "<b><big>우분투를 버전 10.10으로 업그레이드</big></b>"

#: ../DistUpgrade/DistUpgrade.ui.h:12
msgid "Difference between the files"
msgstr "파일 간의 차이점"

#: ../DistUpgrade/DistUpgrade.ui.h:13
msgid "Distribution Upgrade"
msgstr "배포판 업그레이드"

#: ../DistUpgrade/DistUpgrade.ui.h:17
msgid "Restarting the computer"
msgstr "시스템을 재시작하는 중"

#: ../DistUpgrade/DistUpgrade.ui.h:18
msgid "Setting new software channels"
msgstr "소프트웨어 채널을 수정하는 중"

#: ../DistUpgrade/DistUpgrade.ui.h:19
msgid "Terminal"
msgstr "터미널"

#: ../DistUpgrade/DistUpgrade.ui.h:20
msgid "_Cancel Upgrade"
msgstr "업그레이드 취소(_C)"

#: ../DistUpgrade/DistUpgrade.ui.h:21
msgid "_Continue"
msgstr "계속(_C)"

#: ../DistUpgrade/DistUpgrade.ui.h:23
msgid "_Replace"
msgstr "바꾸기(_R)"

#: ../DistUpgrade/DistUpgrade.ui.h:24
msgid "_Report Bug"
msgstr "버그 보고(_R)"

#: ../DistUpgrade/DistUpgrade.ui.h:26
msgid "_Resume Upgrade"
msgstr "업그레이드 계속(_R)"

#: ../DistUpgrade/DistUpgrade.ui.h:27
msgid "_Start Upgrade"
msgstr "업그레이드 시작(_S)"

#: ../UpdateManager/backend/InstallBackendSynaptic.py:54
msgid "Please wait, this can take some time."
msgstr "잠시만 기다리십시오. 이 작업은 약간 시간이 걸립니다."

#: ../UpdateManager/backend/InstallBackendSynaptic.py:56
msgid "Update is complete"
msgstr "업데이트가 끝났습니다."

#: ../UpdateManager/DistUpgradeFetcher.py:85
#: ../UpdateManager/DistUpgradeFetcherKDE.py:103
msgid "Could not find the release notes"
msgstr "릴리즈 정보를 찾을 수 없습니다."

#: ../UpdateManager/DistUpgradeFetcher.py:86
#: ../UpdateManager/DistUpgradeFetcherKDE.py:104
msgid "The server may be overloaded. "
msgstr "서버에 접속자가 너무 많습니다. "

#: ../UpdateManager/DistUpgradeFetcher.py:96
#: ../UpdateManager/DistUpgradeFetcherKDE.py:108
msgid "Could not download the release notes"
msgstr "릴리즈 정보를 다운로드 할 수 없습니다."

#: ../UpdateManager/DistUpgradeFetcher.py:97
#: ../UpdateManager/DistUpgradeFetcherKDE.py:109
msgid "Please check your internet connection."
msgstr "인터넷 연결 상태를 확인하십시오."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:62
#: ../UpdateManager/DistUpgradeFetcherKDE.py:85
msgid "Upgrade"
msgstr "업그레이드"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:89
#: ../data/glade/UpdateManager.ui.h:13
#: ../data/glade/UpgradePromptDialog.ui.h:4
msgid "Release Notes"
msgstr "릴리즈 정보"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:128
#: ../UpdateManager/DistUpgradeFetcherKDE.py:141
#: ../UpdateManager/DistUpgradeFetcherKDE.py:143
msgid "Downloading additional package files..."
msgstr "추가 패키지 파일 다운로드 중..."

#: ../UpdateManager/DistUpgradeFetcherKDE.py:141
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "파일 %s / %s, 속도 %sB/s"

#: ../UpdateManager/DistUpgradeFetcherKDE.py:143
#, python-format
msgid "File %s of %s"
msgstr "파일 %s / %s"

#: ../UpdateManager/ChangelogViewer.py:70
msgid "Open Link in Browser"
msgstr "브라우저로 링크 열기"

#: ../UpdateManager/ChangelogViewer.py:72
msgid "Copy Link to Clipboard"
msgstr "링크를 클립보드에 복사"

#: ../UpdateManager/GtkProgress.py:161
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "%(total)li개 중 %(current)li번째 파일을 %(speed)s/s의 속도로 받는 중"

#: ../UpdateManager/GtkProgress.py:166
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "%(total)li개 중 %(current)li번째 파일을 다운로드 중"

#: ../UpdateManager/UpdateManager.py:326
#, python-format
msgid "Version %s: \n"
msgstr "버전 %s: \n"

#: ../UpdateManager/UpdateManager.py:383
msgid "Downloading list of changes..."
msgstr "변경 사항 목록을 다운로드하고 있습니다..."

#: ../UpdateManager/UpdateManager.py:423
msgid "_Uncheck All"
msgstr "전체 선택 취소(_U)"

#: ../UpdateManager/UpdateManager.py:429
msgid "_Check All"
msgstr "전체 선택(_C)"

#: ../UpdateManager/UpdateManager.py:476
#, python-format
msgid "%(count)s update has been selected."
msgid_plural "%(count)s updates have been selected."
msgstr[0] "%(count)s개의 업데이트가 선택 되었습니다."

#: ../UpdateManager/UpdateManager.py:480
#, python-format
msgid "%s will be downloaded."
msgstr "%s를 다운로드 하게됩니다."

#: ../UpdateManager/UpdateManager.py:491
msgid "The update has already been downloaded, but not installed"
msgid_plural "The updates have already been downloaded, but not installed"
msgstr[0] "업데이트는 이미 다운로드 되었으나 설치되지 않음"

#: ../UpdateManager/UpdateManager.py:495
msgid "There are no updates to install"
msgstr "설치할 업데이트 없음"

#. TRANSLATORS: this allows to switch the order of the count of
#. updates and the download size string (if needed)
#: ../UpdateManager/UpdateManager.py:500
#, python-format
msgid "%(count_str)s %(download_str)s"
msgstr "%(count_str)s %(download_str)s"

#: ../UpdateManager/UpdateManager.py:507
msgid "Unknown download size."
msgstr "다운로드 크기 알 수 없음"

#: ../UpdateManager/UpdateManager.py:518
msgid ""
"It is unknown when the package information was updated last. Please try "
"clicking on the 'Check' button to update the information."
msgstr "패키지 정보가 언제 마지막으로 업데이트 되었는지 알 수 없습니다. 정보를 업데이트하려면 '점검' 버튼을 눌러보십시오."

#: ../UpdateManager/UpdateManager.py:527
#, python-format
msgid "The package information was last updated %(days_ago)s day ago."
msgid_plural ""
"The package information was last updated %(days_ago)s days ago."
msgstr[0] "꾸러미 정보가 %(days_ago)s일 전에 갱신 되었습니다."

#: ../UpdateManager/UpdateManager.py:531
#, python-format
msgid "The package information was last updated %(hours_ago)s hour ago."
msgid_plural ""
"The package information was last updated %(hours_ago)s hours ago."
msgstr[0] "꾸러미 정보가 %(hours_ago)s시간 전에 갱신 되었습니다."

#: ../UpdateManager/UpdateManager.py:535
msgid "The package information was last updated less than one hour ago."
msgstr "패키지 정보가 1시간 이내에 마지막으로 업데이트 되었습니다."

#: ../UpdateManager/UpdateManager.py:555 ../data/glade/UpdateManager.ui.h:23
msgid ""
"Software updates correct errors, eliminate security vulnerabilities and "
"provide new features."
msgstr "소프트웨어 업데이트는 에러를 고치고, 보안 문제를 제거하며 새로운 기능을 제공합니다."

#: ../UpdateManager/UpdateManager.py:575
msgid "Welcome to Ubuntu"
msgstr "우분투의 세계에 오신 것을 환영합니다."

#: ../UpdateManager/UpdateManager.py:576
msgid ""
"These software updates have been issued since Ubuntu was released. If you "
"don't want to install them now, choose \"Update Manager\" from the "
"Administration Menu later."
msgstr ""
"우분투가 출시된 이후에 다음 소프트웨어 업데이트가 나왔습니다. 지금 설치하는 대신 나중에 관리 메뉴에서 \"업데이트 관리자\"를 선택할 "
"수도 있습니다."

#: ../UpdateManager/UpdateManager.py:579
msgid "Software updates are available for this computer"
msgstr "이 컴퓨터에 소프트웨어 업데이트를 설치할 수 있습니다"

#: ../UpdateManager/UpdateManager.py:580
msgid ""
"If you don't want to install them now, choose \"Update Manager\" from the "
"Administration menu later."
msgstr "지금 설치를 원하지 않는다면 나중에도 관리 메뉴에서 \"업데이트 관리자\"를 선택하여 작업 할 수 있습니다."

#: ../UpdateManager/UpdateManager.py:629
#, python-format
msgid ""
"The upgrade needs a total of %s free space on disk '%s'. Please free at "
"least an additional %s of disk space on '%s'. Empty your trash and remove "
"temporary packages of former installations using 'sudo apt-get clean'."
msgstr ""
"업그레이드를 위해서는 '%s' 디스크에 총 %s의 빈 공간이 필요합니다.  최소한 추가로 %s의 디스크 공간을 '%s'에서 남겨주세요.  "
"휴지통을 지우고 'sudo apt-get clean'을 이용해서 이전 설치의 임시 꾸러미들을 지우세요."

#: ../UpdateManager/UpdateManager.py:652
msgid ""
"The computer needs to restart to finish installing updates. Please save your "
"work before continuing."
msgstr "업데이트 설치를 완료하기 위한 재시작이 필요합니다. 진행하기 전에 작업을 저장하시기 바랍니다."

#: ../UpdateManager/UpdateManager.py:704
#: ../UpdateManagerHildon/UpdateManagerHildon.py:114
msgid "Reading package information"
msgstr "패키지 정보를 읽는 중"

#: ../UpdateManager/UpdateManager.py:719
msgid "Connecting..."
msgstr "접속..."

#: ../UpdateManager/UpdateManager.py:729
msgid "You may not be able to check for updates or download new updates."
msgstr "업데이트를 확인할 수 없거나 새 업데이트를 다운받지 못할 수 있습니다."

#: ../UpdateManager/UpdateManager.py:846
msgid "Could not initialize the package information"
msgstr "꾸러미 정보를 초기화할 수 없습니다."

#: ../UpdateManager/UpdateManager.py:847
msgid ""
"An unresolvable problem occurred while initializing the package "
"information.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:\n"
msgstr ""
"꾸러미 정보를 초기화 하는 과정에서 해결 불가능한 문제가 발생했습니다.\n"
"\n"
"다음 오류 메시지를 함께 포함해서 'update-manager' 꾸러미로 이 버그를 보고해 주십시오:\n"

#: ../UpdateManager/UpdateManager.py:871
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
"Please report this bug against the 'update-manager' package and include the "
"following error message:"
msgstr ""
"업그레이드 작업 계산 과정에서 해결 불가능한 문제가 발생했습니다.\n"
"\n"
"다음 오류 메시지를 함께 포함해서 'update-manager' 꾸러미로 이 버그를 보고해 주십시오:"

#: ../UpdateManager/UpdateManager.py:896
msgid " (New install)"
msgstr " (새로 설치)"

#. TRANSLATORS: the b stands for Bytes
#: ../UpdateManager/UpdateManager.py:903
#, python-format
msgid "(Size: %s)"
msgstr "(크기: %s)"

#: ../UpdateManager/UpdateManager.py:905
#, python-format
msgid "From version %(old_version)s to %(new_version)s"
msgstr "버전 %(old_version)s에서 %(new_version)s(으)로"

#: ../UpdateManager/UpdateManager.py:909
#, python-format
msgid "Version %s"
msgstr "버전 %s"

#: ../UpdateManager/UpdateManager.py:924
msgid "Your Ubuntu release is not supported anymore"
msgstr "현재 사용하고 있는 우분투 버전은 더 이상 지원되지 않습니다."

#: ../UpdateManager/UpdateManager.py:925
msgid ""
"You will not get any further security fixes or critical updates. Please "
"Upgrade to a later version of Ubuntu Linux."
msgstr "새로운 보안패치나 중요 업데이트를 더 이상 받을 수 없습니다. 최신 버전의 우분투 리눅스로 업그레이드해 주십시오."

#: ../UpdateManager/UpdateManager.py:932
msgid "Upgrade information"
msgstr "업그레이드 정보"

#: ../UpdateManager/UpdateManager.py:954 ../do-release-upgrade:84
msgid "Release upgrade not possible right now"
msgstr "출시본 업그레이드는 현재 가능하지 않습니다."

#: ../UpdateManager/UpdateManager.py:955 ../do-release-upgrade:85
#, c-format, python-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr "지금 출시본 업그레이드를 할 수 없으므로 나중에 다시 시도하십시오. 서버의 보고: '%s'"

#: ../UpdateManager/UpdateManager.py:957 ../check-new-release-gtk:85
msgid "Downloading the release upgrade tool"
msgstr "배포판 업그레이드 도구를 다운 받는 중입니다."

#: ../UpdateManager/UpdateManager.py:964
#, python-format
msgid "<b>New Ubuntu release '%s' is available</b>"
msgstr "<b>새 우분투 버전인 '%s'(으)로 업그레이드 할 수 있습니다</b>"

#. we assert a clean cache
#: ../UpdateManager/UpdateManager.py:1003
msgid "Software index is broken"
msgstr "소프트웨어 목록이 망가졌습니다."

#: ../UpdateManager/UpdateManager.py:1004
msgid ""
"It is impossible to install or remove any software. Please use the package "
"manager \"Synaptic\" or run \"sudo apt-get install -f\" in a terminal to fix "
"this issue at first."
msgstr ""
"어떤 소프트웨어도 설치하거나 삭제할 수 없습니다. \"시냅틱\"을 사용하거나 터미널에서 \"sudo apt-get install -"
"f\"를 실행하여 이 문제를 먼저 해결하십시오."

#: ../UpdateManagerHildon/UpdateManagerHildon.py:136
msgid "Downloading Package Information"
msgstr "패키지 정보 내려 받는 중"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:137
msgid ""
"The repositories are being checked for new, removed, or updated software "
"packages"
msgstr "새로 추가 혹은 삭제되었거나 업그레이드된 소프트웨어 패키지를 위해 저장소를 체크하는 중입니다."

#: ../UpdateManagerHildon/UpdateManagerHildon.py:145
msgid "Downloading Package Updates"
msgstr "패키지 업데이트 내려 받는 중"

#: ../UpdateManagerHildon/UpdateManagerHildon.py:146
msgid ""
"The selected package updates are being downloaded and installed on the system"
msgstr "선택된 꾸러미의 업데이트를 시스템에 받고 설치합니다."

#: ../UpdateManagerHildon/UpdateManagerHildon.py:152
msgid "run_synaptic called with unknown action"
msgstr "run_synaptic이 알수 없는 동작을 호출하였습니다."

#: ../UpdateManagerText/UpdateManagerText.py:26
msgid "Cancel"
msgstr "취소"

#: ../UpdateManagerText/UpdateManagerText.py:27
msgid "Install"
msgstr "설치"

#: ../UpdateManagerText/UpdateManagerText.py:45
msgid "Building Updates List"
msgstr "업데이트 목록 구성 중"

#: ../UpdateManagerText/UpdateManagerText.py:48
msgid ""
"\n"
"A normal upgrade can not be calculated, please run: \n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"\n"
"일반적인 업그레이드로 추정할 수 없음. 다음을 실행하십시오:\n"
"  sudo apt-get dist-upgrade\n"
"\n"
"\n"
"이것의 원인이 되는 것은:\n"
" * 이전 업그레이드가 완료되지 않음\n"
" * 설치된 소프트웨어 몇 가지에서의 문제\n"
" * 우분투에서 제공하지 않는 비공식 소프트웨어 패키지\n"
" * 프리 릴리즈 버전 우분투의 일상적인 변경"

#: ../UpdateManagerText/UpdateManagerText.py:117
msgid "Downloading changelog"
msgstr "변경사항 내려 받는 중"

#: ../UpdateManager/Core/MyCache.py:128
#, python-format
msgid "Other updates (%s)"
msgstr "다른 업데이트 (%s)"

#: ../UpdateManager/Core/MyCache.py:285
#, python-format
msgid ""
"Changes for the versions:\n"
"%s\n"
"%s\n"
"\n"
msgstr ""
"이 버전의 변경 사항:\n"
"%s\n"
"%s\n"
"\n"

#: ../UpdateManager/Core/MyCache.py:298
msgid "This change is not coming from a source that supports changelogs."
msgstr "이 변경은 changelog를 지원하는 소스에서 온 것이 아닙니다."

#: ../UpdateManager/Core/MyCache.py:304 ../UpdateManager/Core/MyCache.py:324
msgid ""
"Failed to download the list of changes. \n"
"Please check your Internet connection."
msgstr ""
"변경 목록을 받는 데 실패했습니다. \n"
"인터넷 연결을 확인해 보세요."

#: ../UpdateManager/Core/MyCache.py:313
#, python-format
msgid ""
"The changelog does not contain any relevant changes.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"변경 사항 목록에 내용이 없습니다.\n"
"\n"
"변경 사항 목록이 사용 가능하게 되기 전에는 일단 다음 주소를 사용하거나 나중에 다시 시도하십시오.\n"
"http://launchpad.net/ubuntu/+source/%s/%s/+changelog"

#: ../UpdateManager/Core/MyCache.py:318
#, python-format
msgid ""
"The list of changes is not available yet.\n"
"\n"
"Please use http://launchpad.net/ubuntu/+source/%s/%s/+changelog\n"
"until the changes become available or try again later."
msgstr ""
"아직은 변경 사항 목록을 사용할 수 없습니다.\n"
"\n"
"변경 사항 목록이 사용 가능하게 되기 전에는 일단 다음 주소를 사용하거나 나중에 다시 시도하십시오.\n"
"http://launchpad.net/ubuntu/+source/%s/%s/+changelog"

#: ../UpdateManager/Core/UpdateList.py:52
msgid "Failed to detect distribution"
msgstr "배포판 탐지 실패"

#: ../UpdateManager/Core/UpdateList.py:53
#, python-format
msgid "A error '%s' occurred while checking what system you are using."
msgstr "사용중인 시스템을 확인하는 도중에 오류 '%s'(이)가 발생했습니다."

#: ../UpdateManager/Core/UpdateList.py:64
msgid "Important security updates"
msgstr "중요한 보안 업데이트"

#: ../UpdateManager/Core/UpdateList.py:65
msgid "Recommended updates"
msgstr "추천하는 업데이트"

#: ../UpdateManager/Core/UpdateList.py:66
msgid "Proposed updates"
msgstr "제안하는 업데이트"

#: ../UpdateManager/Core/UpdateList.py:67
msgid "Backports"
msgstr "Backports"

#: ../UpdateManager/Core/UpdateList.py:68
msgid "Distribution updates"
msgstr "배포판 업데이트"

#: ../UpdateManager/Core/UpdateList.py:73
msgid "Other updates"
msgstr "기타 업데이트"

#: ../UpdateManager/Core/UpdateList.py:82
msgid "Previous selected"
msgstr "이전 선택"

#: ../data/glade/UpdateManager.ui.h:1
msgid ""
"<b><big>You must check for updates manually</big></b>\n"
"\n"
"Your system does not check for updates automatically. You can configure this "
"behavior in <i>Software Sources</i> on the <i>Updates</i> tab."
msgstr ""
"<b><big>업데이트를 직접 확인해야 합니다.</big></b>\n"
"\n"
"업데이트 확인이 자동으로 되지 않습니다. 이 동작은 <i>소프트웨어 소스</i> 내의 <i>업데이트</i> 탭에서 설정할 수 있습니다."

#: ../data/glade/UpdateManager.ui.h:4
msgid "<big><b>Not all updates can be installed</b></big>"
msgstr "<big><b>설치하지 못한 업데이트가 있습니다</b></big>"

#: ../data/glade/UpdateManager.ui.h:5
msgid "<big><b>Running on battery</b></big>"
msgstr "<big><b>배터리로 동작 중</b></big>"

#: ../data/glade/UpdateManager.ui.h:6
msgid "<big><b>Starting Update Manager</b></big>"
msgstr "<b><big>업데이트 관리자를 시작합니다</big></b>"

#: ../data/glade/UpdateManager.ui.h:7
msgid "Changes"
msgstr "변경 사항"

#: ../data/glade/UpdateManager.ui.h:8
msgid "Chec_k"
msgstr "점검(_k)"

#: ../data/glade/UpdateManager.ui.h:9
msgid "Co_ntinue"
msgstr "계속(_n)"

#: ../data/glade/UpdateManager.ui.h:10
msgid "Description"
msgstr "설명"

#: ../data/glade/UpdateManager.ui.h:11
msgid "Description of update"
msgstr "업데이트 설명"

#: ../data/glade/UpdateManager.ui.h:12
msgid ""
"It&#x2019;s safer to connect the computer to AC power before updating."
msgstr "It&#x2019;s 업데이트전에 컴퓨터를 AC 전원에 연결하시는 것이 안전합니다."

#: ../data/glade/UpdateManager.ui.h:14
msgid ""
"Run a partial upgrade, to install as many updates as possible. \n"
"\n"
"This can be caused by:\n"
" * A previous upgrade which didn't complete\n"
" * Problems with some of the installed software\n"
" * Unofficial software packages not provided by Ubuntu\n"
" * Normal changes of a pre-release version of Ubuntu"
msgstr ""
"최대한 많은 업데이트를 설치하기 위해 부분적인 업데이트를 합니다.\n"
"\n"
"다음과 같은 이유로 시작되었을 수 있습니다:\n"
" * 이전 업그레이드가 완료되지 않았을 경우\n"
" * 설치된 소프트웨어에 문제가 있을 경우\n"
" * 우분투에서 제공되지 않는 비공식적인 소프트웨어 패키지가 있을 경우\n"
" * 정식 배포되지 않은 버전의 우분투에 일반적인 변경이 있을 경우"

#: ../data/glade/UpdateManager.ui.h:21
msgid "Settings..."
msgstr "설정..."

#: ../data/glade/UpdateManager.ui.h:22
#: ../data/glade/UpgradePromptDialog.ui.h:5
msgid "Show progress of individual files"
msgstr "각 파일의 진행 상황 표시"

#: ../data/glade/UpdateManager.ui.h:24
msgid "Starting Update Manager"
msgstr "업데이트 관리자를 시작합니다."

#: ../data/glade/UpdateManager.ui.h:25
msgid "U_pgrade"
msgstr "업그레이드(_p)"

#: ../data/glade/UpdateManager.ui.h:26 ../data/update-manager.desktop.in.h:3
#: ../data/update-manager-hildon.desktop.in.h:3
msgid "Update Manager"
msgstr "업데이트 관리자"

#: ../data/glade/UpdateManager.ui.h:27
msgid "Your system is running on battery. Are you sure you want to continue?"
msgstr "시스템이 배터리로 동작하고 있습니다. 계속 진행하시겠습니까?"

#: ../data/glade/UpdateManager.ui.h:28
msgid "_Hide this information in the future"
msgstr "앞으로 이 정보 숨김(_H)"

#: ../data/glade/UpdateManager.ui.h:29
msgid "_Install Updates"
msgstr "업데이트 설치(_I)"

#: ../data/glade/UpdateManager.ui.h:30
msgid "_Partial Upgrade"
msgstr "부분 업그레이드(_p)"

#: ../data/glade/UpdateManager.ui.h:32
msgid "_Upgrade"
msgstr "업그레이드(_U)"

#: ../data/glade/UpdateManager.ui.h:33
msgid "updates"
msgstr "업데이트"

#: ../data/glade/UpgradePromptDialog.ui.h:1
msgid ""
"<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>새 버전의 우분투가 사용 가능합니다. 업그레이드 하시겠습니까?</b>"

#: ../data/glade/UpgradePromptDialog.ui.h:2
msgid "Ask Me Later"
msgstr "나중에 묻기"

#: ../data/glade/UpgradePromptDialog.ui.h:3
msgid "Don't Upgrade"
msgstr "아니오, 업그레이드 하지 않겠습니다."

#: ../data/glade/UpgradePromptDialog.ui.h:6
msgid "Yes, Upgrade Now"
msgstr "네, 지금 업그레이드 하겠습니다."

#: ../data/glade/UpgradePromptDialog.ui.h:7
msgid ""
"You can upgrade at a later time by opening Update Manager and click on "
"\"Upgrade\"."
msgstr "나중에 업데이트 관리자에서 업그레이드를 클릭하시면 업그레이드 하실 수 있습니다."

#: ../data/glade/UpgradePromptDialog.ui.h:8
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "새로운 우분투 설치를 취소 하셨습니다."

#: ../data/update-manager.desktop.in.h:1
#: ../data/update-manager-hildon.desktop.in.h:1
msgid "Show and install available updates"
msgstr "설치할 수 있는 업데이트를 보여주고, 설치합니다."

#: ../data/update-manager.desktop.in.h:2
#: ../data/update-manager-hildon.desktop.in.h:2
msgid "Software Updates"
msgstr "소프트웨어 업데이트"

#: ../data/update-manager.schemas.in.h:1
msgid "*deprecated* Check for new distribution releases"
msgstr "*deprecated* 새로운 배포판을 확인합니다."

#: ../data/update-manager.schemas.in.h:2
msgid "Auto close the install window after successful install"
msgstr "설치에 성공하면 자동으로 창을 닫습니다."

#: ../data/update-manager.schemas.in.h:3
msgid "First run welcome message"
msgstr "첫 실행 환영 메시지"

#: ../data/update-manager.schemas.in.h:4
msgid ""
"If automatic checking for updates is disabled, you have to reload the "
"channel list manually. This option allows to hide the reminder shown in this "
"case."
msgstr ""
"자동 업데이트 확인 기능을 사용하지 않으면, 채널 목록을 수동으로 다시 읽어야 합니다. 이 옵션은 이 경우에 알림 기능이 나타나지 않도록 "
"합니다."

#: ../data/update-manager.schemas.in.h:5
msgid "If this key is set a first run welcome message will be presented."
msgstr "이 키가 설정되면 첫 실행 환영 메시지가 제공될 겁니다."

#: ../data/update-manager.schemas.in.h:6
msgid ""
"If this key is set the install window will be automatically closed on "
"successful installation."
msgstr "이 키를 설정하면, 설치 후 자동으로 설치 창이 닫힙니다."

#: ../data/update-manager.schemas.in.h:7
msgid ""
"If this key is set the main update list window will show version information "
"(from version to version)."
msgstr "이 키가 설정되면 주 업데이트 목록 창이 (해당 버전에 따라) 버전 정보를 보여줄 것입니다."

#: ../data/update-manager.schemas.in.h:8
msgid ""
"If this key is set, the main update list window will show the package "
"summary before the package name."
msgstr "이 키를 확정하면, 패키지 이름 전에 패키지 요약을 주 업데이트 목록 창에 보여줄 것입니다."

#: ../data/update-manager.schemas.in.h:9
msgid "Remind to reload the channel list"
msgstr "채널 목록을 다시 읽도록 알려줍니다."

#: ../data/update-manager.schemas.in.h:10
msgid "Show details of an update"
msgstr "업데이트의 자세한 정보를 보여줍니다."

#: ../data/update-manager.schemas.in.h:11
msgid "Show package summary before package name in update list"
msgstr "업데이트 목록에서 패키지 이름 전에 패키지 요약을 보여줍니다."

#: ../data/update-manager.schemas.in.h:12
msgid "Show version in update list"
msgstr "업데이트 목록에 버전 표시"

#: ../data/update-manager.schemas.in.h:13
msgid "Stores the size of the update-manager dialog"
msgstr "업데이트 관리자 창의 크기 저장"

#: ../data/update-manager.schemas.in.h:14
msgid ""
"Stores the state of the expander that contains the list of changes and the "
"description"
msgstr "변경 사항과 설명의 목록을 포함하는 확장자의 상태를 저장합니다."

#: ../data/update-manager.schemas.in.h:15
msgid "The window size"
msgstr "창 크기"

#: ../data/update-manager.schemas.in.h:16
msgid ""
"This key is deprecated in favor of the file /etc/update-manager/release-"
"upgrades Check automatically if a new version of the current distribution is "
"available and offer to upgrade (if possible)."
msgstr ""
"이 키는 /etc/update-manager/release-upgrades 파일을 이용하기 위해 더 이상 지원하지 않습니다. 이 키는 "
"현재 배포판의 새로운 버전을 이용할 수 있는지 자동으로 점검하고, (가능하다면) 업그레이드를 하도록 요청합니다."

#: ../data/update-manager.schemas.in.h:17
msgid ""
"This will permanently hide the new release prompt from check-new-release-"
"gtk. Note that the small button in the main update-manager UI will still be "
"there."
msgstr ""
"이 것은 chek-new-release-gtk의 새로운 릴리스 상자를 영구적으로 숨기게 됩니다. 주 update-manager UI내의 "
"작은 버튼은 그대로 남게 됩니다."

#: ../data/update-manager.schemas.in.h:18
msgid "make check-new-release-gtk ignore a given new release"
msgstr "check-new-release-gtk가 새로운 릴리스를 무시하도록 하기"

#: ../update-manager:53 ../do-release-upgrade:33
msgid "Show version and exit"
msgstr "버전을 표시하고 종료"

#: ../update-manager:56
msgid "Directory that contains the data files"
msgstr "데이터 파일이 저장된 디렉터리"

#: ../update-manager:59
msgid "Check if a new Ubuntu release is available"
msgstr "새로운 우분투 버전이 있는지 확인합니다"

#: ../update-manager:62 ../do-release-upgrade:36 ../check-new-release-gtk:136
msgid "Check if upgrading to the latest devel release is possible"
msgstr "최신 개발 버전으로 업그레이드가 가능한지 확인합니다"

#: ../update-manager:66
msgid "Upgrade using the latest proposed version of the release upgrader"
msgstr "배포판 업그레이드 장치의 최종 제안 버전을 사용하여 업그레이드"

#: ../update-manager:73
msgid "Do not focus on map when starting"
msgstr "시작할 때 지도에 포커스를 두지 말 것"

#: ../update-manager:76
msgid "Try to run a dist-upgrade"
msgstr "배포판 업그레이드 실행 시도"

#: ../update-manager:80 ../do-release-upgrade:52
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "샌드박스 aufs 오버레이로 시험 업그레이드"

#: ../update-manager:100
msgid "Running partial upgrade"
msgstr "부분 업그레이드 실행"

#: ../do-release-upgrade:40 ../check-new-release-gtk:140
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr "업그레이드 도구를 이용하여 $distro-proposed에서 최신 배포판으로 업그레이드를 시도합니다"

#: ../do-release-upgrade:44
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"특별 업그레이드 모드에서 실행합니다.\n"
"현재는 데스크탑 시스템의 일반적인 업그레이드를 위한 'desktop' 모드와 서버 시스템을 위한 'server' 모드를 지원합니다."

#: ../do-release-upgrade:50
msgid "Run the specified frontend"
msgstr "지정된 프론트엔드를 실행"

#: ../do-release-upgrade:55
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr "새로운 배포판이 있을 경우에만 확인하기. 결과는 종료 코드를 통해 반환"

#: ../do-release-upgrade:69
msgid "Checking for a new ubuntu release"
msgstr "새로운 우분투 버전을 확인합니다."

#: ../do-release-upgrade:79
msgid "No new release found"
msgstr "새로운 릴리즈가 없습니다."

#: ../do-release-upgrade:91
#, c-format
msgid "New release '%s' available."
msgstr "새 배포판 '%s'(이)가 사용가능합니다."

#: ../do-release-upgrade:92
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "업그레이드하기 위해 'do-release-upgrade'를 실행하세요."

#: ../check-new-release-gtk:65
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "우분투 %(version)s 업그레이드 가능합니다."

#: ../check-new-release-gtk:101
#, c-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "우분투 %s 로 업그레이드 하는것을 취소 하셨습니다."

#: ../Janitor/computerjanitor/exc.py:30
#, python-format
msgid "Unimplemented method: %s"
msgstr "미구현된 방법: %s"

#: ../Janitor/computerjanitor/file_cruft.py:46
msgid "A file on disk"
msgstr "디스크에 있는 파일"

#: ../Janitor/computerjanitor/package_cruft.py:45
msgid ".deb package"
msgstr ".deb 패키지"

#: ../Janitor/computerjanitor/missing_package_cruft.py:33
msgid "Install missing package."
msgstr "빠진 패키지 설치."

#: ../Janitor/computerjanitor/missing_package_cruft.py:42
#, python-format
msgid "Package %s should be installed."
msgstr "%s 패키지가 반드시 설치되어야 합니다."

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:36
#, python-format
msgid "%i obsolete entries in the status file"
msgstr "%i는 상태 파일에서 안 쓰게 된 엔트리"

#: ../Janitor/plugins/dpkg_status_plugin.py:39
msgid "Obsolete entries in dpkg status"
msgstr "dpkg 상태에서 안 쓰게 된 엔트리"

#. pragma: no cover
#: ../Janitor/plugins/dpkg_status_plugin.py:42
msgid "Obsolete dpkg status entries"
msgstr "안 쓰게 된 dpkg 상태 엔트리"

#: ../Janitor/plugins/kdelibs4to5_plugin.py:39
msgid ""
"When upgrading, if kdelibs4-dev is installed, kdelibs5-dev needs to be "
"installed. See bugs.launchpad.net, bug #279621 for details."
msgstr ""
"업그레이드 시 kdelibs4-dev가 설치되어 있다면 kdelibs5-dev를 설치해야 합니다. bugs.launchpad.net의 "
"bug #279621에서 자세한 사항을 볼 수 있습니다."

#: ../Janitor/plugins/langpack_manual_plugin.py:34
#, python-format
msgid "%s needs to be marked as manually installed."
msgstr "%s는 수동 설치로 표시될 필요가 있습니다."

#: ../Janitor/plugins/remove_lilo_plugin.py:28
msgid ""
"Remove lilo since grub is also installed.(See bug #314004 for details.)"
msgstr "grub이 설치되어 있으므로 lilo는 제거하세요.(자세한 내용은 버그 #314004를 참조.)"

#, python-format
#~ msgid "The package information was last updated %s hour ago."
#~ msgid_plural "The package information was last updated %s hours ago."
#~ msgstr[0] "패키지 정보가 %s시간 전에 마지막으로 업데이트 되었습니다."

#, python-format
#~ msgid "The package information was last updated %s day ago."
#~ msgid_plural "The package information was last updated %s days ago."
#~ msgstr[0] "패키지 정보가 %s일 전에 마지막으로 업데이트 되었습니다."

#~ msgid "Unknown download size"
#~ msgstr "다운로드 크기를 알 수 없음"

#, python-format
#~ msgid "<b>Downgrade %s</b>"
#~ msgstr "<b>%s 다운그레이드</b>"

#~ msgid "Support for some applications ended"
#~ msgstr "일부 프로그램의 지원이 종료되었습니다"

#~ msgid ""
#~ "The upgrade is now aborted. Your system could be in an unusable state. A "
#~ "recovery will run now (dpkg --configure -a)."
#~ msgstr ""
#~ "지금 업그레이드가 중단되었습니다.  시스템이 사용불가 상태일 수 있습니다.  지금 복구가 실행될 것입니다 (dpkg --configure "
#~ "-a)."

#~ msgid ""
#~ "The upgrade is now aborted. Please check your Internet connection or "
#~ "installation media and try again. All files downloaded so far are kept."
#~ msgstr ""
#~ "지금 업그레이드가 중단되었습니다.  인터넷 연결이나 설치 미디어를 확인하고 다시 시도해 보십시요.  지금까지 다운로드된 모든 파일들은 "
#~ "보존됩니다."

#, python-format
#~ msgid ""
#~ "The upgrade is now aborted. The upgrade needs a total of %s free space on "
#~ "disk '%s'. Please free at least an additional %s of disk space on '%s'. "
#~ "Empty your trash and remove temporary packages of former installations using "
#~ "'sudo apt-get clean'."
#~ msgstr ""
#~ "지금 업그레이드가 중단되었습니다.  업그레이드는 '%s' 디스크에 총 %s의 여유 공간이 필요합니다.  '%s'에 추가로 최소 %s의 "
#~ "디스크 공간이 더 확보되어야 합니다.  휴지통을 비우고 'sudo apt-get clean'을 사용해서 이전 설치의 임시 패키지들을 "
#~ "삭제하십시요."

#~ msgid ""
#~ "The upgrade is now aborted. Please check your Internet connection or "
#~ "installation media and try again. "
#~ msgstr "지금 업그레이드가 중단되었습니다.  인터넷 연결이나 설치 미디어를 확인하고 다시 시도해 보십시요. "

#~ msgid ""
#~ "Please report this as a bug (if you haven't already) and include the files "
#~ "/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
#~ "report. The upgrade is now aborted.\n"
#~ "Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
#~ msgstr ""
#~ "이것을 버그로 보고해주시고, 보고서에 /var/log/dist-upgrade/main.log 파일과 /var/log/dist-"
#~ "upgrade/apt.log 파일을 첨부해 주십시오.  업그레이드는 지금 중단되었습니다.\n"
#~ "sources.list 원본은 /etc/apt/sources.list.distUpgrade에 저장되었습니다."

#~ msgid ""
#~ "Please report this as a bug and include the files /var/log/dist-"
#~ "upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
#~ "upgrade is now aborted.\n"
#~ "Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
#~ msgstr ""
#~ "이것을 버그로 보고해주시고, 보고서에 /var/log/dist-upgrade/main.log 파일과 /var/log/dist-"
#~ "upgrade/apt.log 파일을 첨부해 주십시오.  업그레이드는 지금 중단되었습니다.\n"
#~ "sources.list 원본은 /etc/apt/sources.list.distUpgrade에 저장되었습니다."

#~ msgid "Restart _Later"
#~ msgstr "나중에 재시작(_L)"

#~ msgid "<b><big>Upgrading Ubuntu to version 10.04 LTS</big></b>"
#~ msgstr "<b><big>우분투 10.04 LTS 버전으로 업그레이드 중입니다.</big></b>"

#, python-format
#~ msgid ""
#~ "Download size: %s\n"
#~ "%s selected."
#~ msgstr ""
#~ "다운로드 크기: %s\n"
#~ "%s 선택됨."

#~ msgid ""
#~ "Canonical Ltd. no longer provides support for the following software "
#~ "packages. You can still get support from the community.\n"
#~ "\n"
#~ "These packages will be suggested for removal at the end of the upgrade "
#~ "because you have not enabled the community maintained software channel "
#~ "(universe)."
#~ msgstr ""
#~ "캐노니컬사(Canonical Ltd.)는 다음의 소프트웨어 패키지 지원을 더이상 제공하지 않습니다. 하지만 커뮤니티로부터 지원을 받을 수 "
#~ "있습니다.\n"
#~ "커뮤니티에서 유지하는 소프트웨어 채널(universe)를 활성화하지 않았기 때문에 이 패키지들을 제거하시길 제안합니다."

#~ msgid ""
#~ "Canonical Ltd. no longer provides support for the following software "
#~ "packages. You can still get support from the community."
#~ msgstr ""
#~ "캐노니컬사(Canonical Ltd.)는 다음의 소프트웨어 패키지 지원을 더이상 제공하지 않습니다. 하지만 커뮤니티로부터 지원을 받을 수 "
#~ "있습니다."
